core/core.md

```diff
@@ -62,14 +62,16 @@ Thus the Fora protocol must assume that malicious actors will never reach the th
 
```

Comment:
We should also add a section here in `Community Governance Security Model` discussing the Security Model in light of this change. The above security model holds from the individual voter perspective.

We should have a similar point regarding how much Admittor "weight" must be byzantine before the community is no longer secure.

Reply:
sounds good

Reply:
The first paragraph does a good job explaining the concept abstractly. I added to the second paragraph. lmk how this looks